### ISSUE TITLE
Heroic won't load games: Cannot destructure property 'namespace' of 'asset_info' as it is undefined.

### ISSUE THREAD

**Issue by cjlaborde** [Heroic-Games-Launcher/HeroicGamesLauncher#790]
Stuck in Loading, I am using Pop OS 20 LTS and install it using the .deb file.

```
▶ heroic
INFO: Trying to get user information
INFO: Trying to get user information
INFO: Trying to get user information
WARNING: Protocol already registered.
[960568:1217/032510.221092:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
WARNING: python command not found
INFO: Command 'python3' found. Version: '3.8.10'
INFO: Frontend: Refreshing Library
INFO: Refreshing library...
INFO: Legendary: Refreshing Epic Games...
INFO: Refreshing Epic Games...
[cli] INFO: Logging in...

[Core] INFO: Trying to re-use existing login session...

[cli] INFO: Getting game list... (this may take a while)

ERROR: TypeError: Cannot destructure property 'namespace' of 'asset_info' as it is undefined.
Error occurred in handler for 'refreshLibrary': TypeError: Cannot read properties of null (reading 'title')
    at /opt/Heroic/resources/app.asar/build/legendary/library.js:123:29
    at Array.sort (<anonymous>)
    at LegendaryLibrary.getGames (/opt/Heroic/resources/app.asar/build/legendary/library.js:122:55)
    at async /opt/Heroic/resources/app.asar/build/main.js:455:12
    at async node:electron/js2c/browser_init:201:563
(node:960568) UnhandledPromiseRejectionWarning: TypeError: Cannot destructure property 'namespace' of 'asset_info' as it is undefined.
    at LegendaryLibrary.loadFile (/opt/Heroic/resources/app.asar/build/legendary/library.js:269:17)
    at /opt/Heroic/resources/app.asar/build/legendary/library.js:383:41
    at Array.map (<anonymous>)
    at LegendaryLibrary.loadAll (/opt/Heroic/resources/app.asar/build/legendary/library.js:383:18)
    at /opt/Heroic/resources/app.asar/build/legendary/library.js:75:18
    at async LegendaryLibrary.getGames (/opt/Heroic/resources/app.asar/build/legendary/library.js:109:17)
    at async /opt/Heroic/resources/app.asar/build/main.js:455:12
    at async node:electron/js2c/browser_init:201:563
(Use `heroic --trace-warnings ...` to show where the warning was created)
(node:960568) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
ERROR: Frontend: Error: Error invoking remote method 'refreshLibrary': TypeError: Cannot read properties of null (reading 'title')
INFO: Frontend: No cache found, getting data from legendary...
INFO: Trying to get user information
INFO: Refreshing library...
ERROR: TypeError: Cannot destructure property 'namespace' of 'asset_info' as it is undefined.
Error occurred in handler for 'readConfig': TypeError: Cannot read properties of null (reading 'title')
    at /opt/Heroic/resources/app.asar/build/legendary/library.js:123:29
    at Array.sort (<anonymous>)
    at LegendaryLibrary.getGames (/opt/Heroic/resources/app.asar/build/legendary/library.js:122:55)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /opt/Heroic/resources/app.asar/build/main.js:422:20
    at async node:electron/js2c/browser_init:201:563
INFO: Trying to get user information
INFO: checking for heroic updates

```

**Comment by flavioislima** [comment]
Yes, I had this issue as well and I fixed it already in the code so should be fixed on next version.
I'm not sure if this was introduced in the latest legendary versions or if Epic changed something on their metadata files, but they removed this `namespace` information we used to check if it's a game or a UE asset.
But I found a workaround and pushed to the main branch.
For now, try updating Legendary by downloading it from their github and selecting the binary on Heroic settings.
I'm not sure if this will fix it thought. 
I shall release a new version this weekend with a fix anyway.
